### DISCUSSION TITLE
Release script update

### DISCUSSION THREAD

**Discussion by gastaldi** [quarkiverse/quarkiverse#204] [Team Posts]
Hello!

In order to simplify and to minimize the number of actions used in the release.yml workflow, I created some PRs (linked in https://github.com/quarkiverse/quarkiverse-devops/issues/210) updating the existing scripts to our latest recommended script.

However, after these PRs were created, I realized that some repositories are already building and releasing using JDK 17 but the PR (in a wrong way) changes that to JDK 11. 

Although I have already fixed some PRs,  I might have missed some, so I'd advise to please review these PRs before merging and make sure that the correct JDK is set before merging.

Thank you!

**Comment by zhfeng** [comment]
Thanks @gastaldi - there is no such PR for [quarkus-pooled-jms](https://github.com/quarkiverse/quarkus-pooled-jms/), did you miss this repository?

**Reply by gastaldi** [reply]
I've interrupted the script once I found that it was overriding JDK 17 with JDK 11. Let me run for that repository now
